op-scim-bridge/values.yaml

```diff
@@ -17,16 +17,18 @@ scim:
     resources:
       requests:
         storage: 1Gi
-    # storageClass for the volume. do-block-storage is recommended for Digital Ocean
+    # storageClass for the volume. do-block-storage is recommended for Digital Ocean. Set to "-" to set storageClass to "".
```

Comment:
```suggestion
    # storageClass for the volume. do-block-storage is recommended for Digital Ocean. Set to "-" to set storageClass to "" to use the platform default.
```

Reply:
This is not entirely accurate and a bit confusing, but you set the `storageClass` to `""` to _not_ use the default.

> "A PVC with its storageClassName set equal to "" is always interpreted to be requesting a PV with no class, so it can only be bound to PVs with no class (no annotation or one set equal to "")."

From the Kubernetes PersistantVolumeClaim Class [documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1).